content/source/docs/enterprise/private/minio-setup-guide.html.md

```diff
@@ -0,0 +1,95 @@
+
+## Overview
+
+This guide will walk through installing minio in a Docker container alongside PTFE on the same host, with PTFE configured in the "Production - External Services" [operational mode](.install-installer.html#operational-mode-decision).  This is not a production-ready configuration: it's intended to guide you to a working configuration that can later be automated and hardened.  Data will not be persisted outside of an ephemeral Docker volume, minio will not start on system boot, etc.  It is assumed your instance will have access to the Internet and that you will be performing an online install of PTFE.
+
+## Installation
+
+### System preparation
+
+Ensure your Linux instance meets the [requirements](./install-installer.html#linux-instance).  You will need [jq](https://stedolan.github.io/jq/) and the [AWS cli](https://aws.amazon.com/cli/).
+
+You also need a PostgreSQL database that meets the [requirements](.install-installer.html#postgresql-requirements), as this is part of the external services operational mode.
+
+### PTFE installation
+
+Begin with an [online installation](./install-installer.html#run-the-installer-online).  Once the installation script has finished and you're presented with the following text, move on to the next section:
+
+```
+To continue the installation, visit the following URL in your browser:
+
+  https://<this_server_address>:8800
+```
+
+### Start minio
+
+Now you'll start the minio container, mounting a volume so that you can gain access to the generated config:
+
+    docker run \
+      -d \
+      --name minio \
+      -v /run/minio/config:/root/.minio minio/minio:latest \
+      -- \
+      server /data
+
+Ensure that minio has started by watching for `/var/run/minio/config/config.json` to be written:
+
+    while [ ! -e /var/run/minio/config/config.json ]; do
+      sleep 3
+    done
+
+You now need to collect several pieces of information about your running minio instance:
+
+- the IP address of the running container: `docker inspect minio | jq -r .[0].NetworkSettings.IPAddress`
+- the access key: `jq -r .credential.accessKey /var/run/minio/config/config.json`
+- the secret key: `jq -r .credential.secretKey /var/run/minio/config/config.json`
+
+### Create a bucket
+
+Like S3, minio does not automatically create buckets.  Use the AWS cli to create a bucket named `ptfe` that will be used to store data:
+
+```bash
+export AWS_ACCESS_KEY_ID="<access key from above>"
+export AWS_SECRET_ACCESS_KEY="<secret key from above>"
+
+aws --region us-east-1 --endpoint-url http://<ip address from above>:9000 s3 mb s3://ptfe
+```
+
+### PTFE installation
+
+You may now [continue the installation in the browser](./install-installer.html#continue-installation-in-browser).
+
+1. provide a valid hostname
+2. choose the "Production" installation type
+3. choose the "External Services" production type
+4. provide the required Database URL for the PostgreSQL configuration
+5. choose "S3" for object storage
```

Comment:
This looks like it might be duplicating other info. Is it reasonable to refer to the info and then instruct the user on where to drop the Minio-specific process in?